### PR TITLE
HDDS-6192. feature/Observability.md translated to Chinese

### PR DESCRIPTION
**Pull request by steinsgateted**
## What changes were proposed in this pull request?

add Observability.zh.md

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-6192

## How was this patch tested?

local hugo test

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 提供了多种工具来获取有关集群当前状态的更多信息。
+
+## Prometheus
+Ozone 原生支持 Prometheus 集成。所有内部指标（由 Hadoop 指标框架收集）都发布在 `/prom` HTTP 端点下。（例如，在 SCM 的 http://localhost:9876/prom 下）。
+
+Prometheus 端点默认打开，但可以通过`hdds.prometheus.endpoint.enabled`配置变量关闭。
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:31]:
```suggestion
Prometheus 端点默认是打开的，但可以通过`hdds.prometheus.endpoint.enabled`配置变量把它关闭。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 提供了多种工具来获取有关集群当前状态的更多信息。
+
+## Prometheus
+Ozone 原生支持 Prometheus 集成。所有内部指标（由 Hadoop 指标框架收集）都发布在 `/prom` HTTP 端点下。（例如，在 SCM 的 http://localhost:9876/prom 下）。
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:29]:
```suggestion
Ozone 原生支持 Prometheus 集成。所有内部指标（由 Hadoop 指标框架收集）都发布在 `/prom`的 HTTP 端点下。（例如，在 SCM 的 http://localhost:9876/prom）。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+Prometheus 端点默认打开，但可以通过`hdds.prometheus.endpoint.enabled`配置变量关闭。
+
+在安全环境中，该页面由 Prometheus 不支持的 SPNEGO 身份验证保护。要在安全环境中启用监控，可以配置特定的身份验证令牌
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:33]:
```suggestion
在安全环境下，该页面是用 SPNEGO 认证来保护的，但 Prometheus 不支持这种认证。为了在安全环境中启用监控，可以配置一个特定的认证令牌。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+`ozone-site.xml` 范例：
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:35]:
```suggestion
`ozone-site.xml` 配置示例：
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+  - job_name: ozone
+    bearer_token: <putyourtokenhere>
+    metrics_path: /prom
+    static_configs:
+     - targets:
+         - "127.0.0.1:9876" 
+```
+
+## 分布式追踪
+分布式追踪可以通过可视化端到端性能来帮助理解性能瓶颈。
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:55]:
```suggestion
分布式追踪可以通过可视化端到端的性能来帮助了解性能瓶颈。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+scrape_configs:
+  - job_name: ozone
+    bearer_token: <putyourtokenhere>
+    metrics_path: /prom
+    static_configs:
+     - targets:
+         - "127.0.0.1:9876" 
+```
+
+## 分布式追踪
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:54]:
```suggestion
## 分布式跟踪
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+分布式追踪可以通过可视化端到端性能来帮助理解性能瓶颈。
+
+Ozone 使用 [jaeger](https://jaegertracing.io) 跟踪库收集跟踪，可以将跟踪数据发送到任何兼容的后端(Zipkin，…)。
+
+跟踪在默认情况下是关闭的，但是可以从 `ozone-site.xml` 打开 `hdds.tracing.enabled`
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:59]:
```suggestion
默认情况下，跟踪功能是关闭的，但可以通过`ozon-site.xml`的`hdds.tracing.enabled`配置变量打开。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+
+```XML
+<property>
+   <name>hdds.tracing.enabled</name>
+   <value>true</value>
+</property>
+```
+
+Jager客户端可以用环境变量进行配置，如[这里](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)所记录的：
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:68]:
```suggestion
Jaeger 客户端可以用环境变量进行配置，如[这份](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)文档所述。
```
There is a typo here in the English document for "Jager", can you help to fix it?
`Jager` -> `Jaeger`

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+
+例如：
+```shell
+JAEGER_SAMPLER_PARAM=0.01
+JAEGER_SAMPLER_TYPE=probabilistic
+JAEGER_AGENT_HOST=jaeger
+```
+
+此配置将记录1%的请求，以限制性能开销。有关jaeger抽样的更多信息，请查看[文档](https://www.jaegertracing.io/docs/1.18/sampling/#client-sampling-configuration)
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:77]:
```suggestion
此配置将记录1%的请求，以限制性能开销。有关 Jaeger 抽样的更多信息，请查看[文档](https://www.jaegertracing.io/docs/1.18/sampling/#client-sampling-configuration)。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+
+## ozone insight
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:79]:
```suggestion
## Ozone Insight
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+ozone insight 是一款 swiss-army-knife 工具，用于检查 Ozone 集群的当前状态。它可以显示特定组件的日志记录、指标和配置。
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:80]:
```suggestion
Ozone Insight 是一个用于检查 Ozone 集群当前状态的工具。它可以显示特定组件的日志记录、指标和配置。
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+
+要检查可用的成分，请使用 `ozone insight list`：
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:82]:
```suggestion
请使用`ozone insight list`命令检查可用的组件：
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+`ozone insights logs` 可以连接到所需的服务并显示与一个特定组件相关的DEBUG/TRACE日志。例如，显示RPC消息：
+
+```shell
+>ozone insight logs om.protocol.client
+
+[OM] 2020-07-28 12:31:49,988 [DEBUG|org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB|OzoneProtocolMessageDispatcher] OzoneProtocol ServiceList request is received
+[OM] 2020-07-28 12:31:50,095 [DEBUG|org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB|OzoneProtocolMessageDispatcher] OzoneProtocol CreateVolume request is received
+```
+
+使用 `-v` 标志，也可以显示 protobuf 信息的内容（TRACE级日志）：
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:169]:
```suggestion
使用 `-v` 标志，也可以显示 protobuf 信息的内容（TRACE级别的日志）：
```

File: hadoop-hdds/docs/content/feature/Observability.zh.md [hunk 14]
```diff
@@ -0,0 +1,215 @@
...
+cmdType: CreateVolume
+traceID: ""
+success: false
+message: "Volume already exists"
+status: VOLUME_ALREADY_EXISTS
+```
+
+<div class="alert alert-warning" role="alert">
+
+在 hood 下，`ozone insight` 使用 HTTP 端点来检索所需的信息（`/conf`、`/prom`和`/logLevel`端点）。它在安全环境中还不被支持。
```
Comment (cku328) [inline hadoop-hdds/docs/content/feature/Observability.zh.md:213]:
```suggestion
实际上 `ozone insight` 是通过 HTTP 端点来检索所需的信息（`/conf`、`/prom`和`/logLevel`端点），它在安全环境中还不被支持。
```

**Review by cku328 — CHANGES_REQUESTED** [review]
Thanks to @steinsgateted for translating this document.
Welcome to the Ozone Community!

I have revised some of the content to make it smoother.
If you have any questions, please feel free to discuss.

**Review by cku328 — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @steinsgateted for the translation, @cku328 for the review.  Can we merge it or is it waiting for further review?

**Comment by cku328** [comment]
> Thanks @steinsgateted for the translation, @cku328 for the review. Can we merge it or is it waiting for further review?

Thanks @adoroszlai for the comment.
I think so, let me merge it.

**Pull request merged**